src/pytest_cov/plugin.py

```diff
@@ -226,6 +244,13 @@ def pytest_runtestloop(self, session):
                 compat_session.testsfailed += 1
 
     def pytest_terminal_summary(self, terminalreporter):
+        if self._disabled:
+            markup = {'red': True, 'bold': True}
+            msg = (
+                'WARNING: Coverage disabled by user!'
```

Comment:
Also the reason for not mentioning the no-cov flag in there is precisely to avoid spaghetti coding. You would constantly have to maintain this line whenever you decide to rename this variable. Unfortunate enough already that I have to copy it in the README.rst I know that Ctrl+R exists but some ppl. prefer typing. Not to mention --no-cov vs. no_cov. By all means I will make the changes but wanted you to know the reason just for the record.